### PR TITLE
devel: add another potential ip to the list

### PR DESCRIPTION
**Pull request by david-caro**

### CODE REVIEW

**Comment by david-caro** [comment]
This is mostly so I don't have to change that every time on my dev environment, we might want to come up with a more "generic and automated" way of doing so at some point, but probably after moving to helmfile as that might open new solutions

File: deploy/devel/deployment.patch.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 - op: replace
   path: /spec/template/spec/containers/0/env/1/value
   # this resolves to the ip of the host running minikube
```
Comment (dhinus) [inline deploy/devel/deployment.patch.yaml]: can we update the comment here?

**Review by dhinus — CHANGES_REQUESTED** [review]

File: deploy/devel/deployment.patch.yaml (hunk 1 above)
Comment (david-caro) [reply deploy/devel/deployment.patch.yaml]: Sure, what do you want mentioned there?

Comment (dhinus) [reply deploy/devel/deployment.patch.yaml]: Something like

```
This patches the ALLOWEDDOMAINS env variable to allow connections from all the hostnames and IPs currently used in dev environments.
TODO: make this more generic (switching to Helm might help)
```

Feel free to rewrite as it sounds best to you!

**Review by dhinus — APPROVED** [review]

**Pull request merged**